examples/fabric/connection/example_port_to_aws/go/go.mod

```diff
@@ -2,7 +2,7 @@ module equinix-fabric-connection-example_port_to_aws
 
 go 1.21
 
-toolchain go1.22.6
+toolchain go1.22.0
```

Comment:
This is _closer_ than what CI used in #216 but still not quite right.  This is using the Go version from `provider/go.mod`, so maybe this change is correct?  Where did 1.22.6 come from initially?